assessment_software/src/app/table-component/table.component.ts

```diff
@@ -8,15 +8,10 @@ styleUrls:['./table.component.css']
 export class TableComponent{
     @Input() headers:string[]=[]; 
     @Input() width:Number=700;
-    @Input() data: {[key: string]: string}[]=[{
-      Name:"",
-      ID:"",
-      Passcode:"",
-      Email:"",
-     }
-   ];
-   @Input() wrapStyle:string='';
-   @Input() scrollStyle:string='';
+    @Input() data: {[key: string]: string}[]=[{ }];
+   @Input() wrapStyle:boolean=false;
```

Comment:
WrapStyle and scrollStyle is not a good naming. The naming should hint the data type. From wrapStyle, you can't tell if it is boolean or not. Bur, naming like wraps or isWrappable would be a better naming.